### PR TITLE
Course completed event creation

### PR DESCRIPTION
**Pull request by caperneoignis**
This resolves the course completed event creation not being done. as stated in issue #36. Please let me know if any changes need to be made.

### CODE REVIEW

**Comment by ryansmith94** [comment]
Looks ok to me @jlowe64, however, there does need to be some integration test written in the [plugin](https://github.com/xAPI-vle/moodle-logstore_xapi) @caperneoignis.

File: src/Controller.php [hunk 1]
```diff
@@ -39,7 +40,6 @@ class Controller extends PhpObj {
         '\core\event\user_created' => 'UserRegistered',
         '\core\event\user_enrolment_created' => 'EnrolmentCreated',
         '\mod_scorm\event\sco_launched' => 'ScormLaunched',
-        '\mod_feedback\event\response_submitted' => ['FeedbackSubmitted','FeedbackQuestionSubmitted'],
```
Comment (ryansmith94) [inline src/Controller.php]: This needs to be reverted.

**Comment by ryansmith94** [comment]
@jlowe64 I've made a comment on the code that needs to be resolved before merging.

**Comment by caperneoignis** [comment]
@ryansmith94 OK, so write a unit test for course completed in the plug in? Let me see how that's done, and I'll ask any questions if they come up. I'll do a pull request once that's done.

**Comment by ryansmith94** [comment]
@caperneoignis yeah that's correct, you should be able to see a lot of similar tests in there. Yeah feel free to ask any questions. Thanks 😄

**Pull request merged**